### PR TITLE
better integration of digits and numbers

### PR DESCRIPTION
**Pull request by fidgetingbits**
this solves the problem described in the code comment. originally aegis proposed that i switch it so that the existing `number` rule contained the new `digits` references, but it seems like that is a problem because of the way the existing scaled function call is done, since we have no easy way to differentiate the `digits` digits and actual `number` digits passed in m. so i opted to change the existing `numbers` rule into a new name. i'm not sure how to create a name that isn't part of the `user.` object prefix, because trying new create one without it would always say that it wasn't defined, which is why it became `user.number_scaled` for now.

anyways i'm happy to change this if you have ideas but definitely way easier to use this number method than the existing one because you can speak numbers much more naturally

### CODE REVIEW

**Comment by fidgetingbits** [comment]
originally I accidentally left a testing bit with `digits` in `numbers_scalled`, so just removed it. also i've been using an python pep8-ish fixer (called black) and the last commit includes those cleanups so I guess this is a good opportunity to find out if you agree that using this for commits is okay. to me it makes coding with voice a lot easier, because i have to do less work cleaning up spacing, etc. but it will occasionally make the commits a bit more involved.

**Comment by knausj85** [comment]
Thanks, this has been on my list for awhile.

aegis is the only one that can declare things without “user.” if you ever want a capture or action declared by talon, you can ask him nicely.

I’ll tinker with this a bit, as I would like to see it consolidated into numbers.

File: code/numbers.py [hunk 1]
```diff
@@ -116,13 +116,29 @@ def number_small(m):
...
+@ctx.capture('self.number_scaled', rule=f'(<digits> | [<digits>] <number_small> [{alt_scales} ([and] (<number_small> | {alt_scales} | <number_small> {alt_scales}))*])')
+def number_scaled(m):
     return fuse_num(fuse_scale(fuse_num(fuse_scale(list(m), 3))))[0]
 
+# This rule offers more colloquial number speaking when combined with a command
+# like: "go to line <number>"
+# Example: " one one five            " == 115
+#          " one fifteen             " == 115
+#          " one hundred and fifteen " == 115
+@ctx.capture('number', rule=f'(<digits> | [<digits>] <user.number_scaled>)$')
```
Comment (knausj85) [inline code/numbers.py]: Is the $ necessary here?

Comment (fidgetingbits) [reply code/numbers.py]: I removed it. originally i was using that to test what seemed like a time out between saying digits in the actual scaled number, but i think it was due to the new version of town having an overly aggressive time out

File: code/numbers.py [hunk 2]
```diff
@@ -116,13 +166,37 @@ def number_small(m):
...
-@ctx.capture('number_signed', rule=f'[negative] <number>')
+
+# This rule offers more colloquial number speaking when combined with a command
+# like: "go to line <number>"
+# Example: " one one five            " == 115
+#          " one fifteen             " == 115
+#          " one hundred and fifteen " == 115
+@ctx.capture("number", rule=f"(<digits> | [<digits>] <user.number_scaled>)")
+def number(m):
+    return "".join(str(i) for i in list(m))
```
[the remark this answers is not in the document]
Comment (fidgetingbits) [reply code/numbers.py:185]: sounds good. i'll change that and test in the next day or two hopefully

[the remark this answers is not in the document]
Comment (fidgetingbits) [reply code/numbers.py]: done

**Pull request merged**